### ISSUE TITLE
project/jni/sdl-1.3/include/SDL_screenkeyboard.h is corrupt

### ISSUE THREAD

**Issue by rmtew** [pelya/commandergenius#9]
This change:

https://github.com/pelya/commandergenius/commit/04a43755658d052eb660ce0d0656e04ba49e620d

Clobbered the file with:

../../sdl-1.2/include/SDL_screenkeyboard.h

**Comment by rmtew** [comment]
msysgit problem

**Comment by pelya** [comment]
I've moved SDL_screenkeyboard.h from sdl-1.3 to sdl-1.2 include directory, and put a symlink to it from sdl-1.3.
Anyway, you should either use Git command from inside Cygwin, which supports symlinks (the msys version or TortoiseGit do not support them), or use a proper Linux environment, such as Portable Ubuntu.

**Comment by rmtew** [comment]
I've come to realise that this is a problem with symlink support on
Windows in general.  Even with working symlinks (Windows mklink
command), there are strange compilation errors where the file cannot
be found:

http://forums.gibberlings3.net/index.php?showtopic=24504&view=findpost&p=202619

I tried cygwin and it had exactly the same problems as msys, for me.

Anyway, I've since gotten commandergenius to work on Windows with the
steps given in the above thread, within a normal MingW/msys
installation and with symlinked files replaced with copies - something
I had to do manually.

On Tue, Jul 10, 2012 at 2:18 AM, pelya
reply@reply.github.com
wrote:
